suite-native/device-manager/src/components/WalletItem.tsx

```diff
@@ -0,0 +1,71 @@
+import { Pressable } from 'react-native';
+import { useSelector } from 'react-redux';
+
+import { HStack, Radio, Text } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { Icon } from '@suite-common/icons';
+import { TrezorDevice } from '@suite-common/suite-types';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { selectDevice } from '@suite-common/wallet-core';
+
+const walletItemStyle = prepareNativeStyle<{ isSelected: boolean }>((utils, { isSelected }) => ({
+    paddingHorizontal: utils.spacings.medium,
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    height: 60,
+    gap: 12,
+    backgroundColor: utils.colors.backgroundSurfaceElevation1,
+    borderWidth: utils.borders.widths.small,
+    borderRadius: 12,
+    borderColor: utils.colors.borderElevation1,
+    extend: {
+        condition: isSelected,
+        style: {
+            borderWidth: utils.borders.widths.large,
+            borderColor: utils.colors.borderSecondary,
+        },
+    },
+}));
+
+type WalletItemProps = {
+    device: TrezorDevice;
+    isSelectable?: boolean;
+    onPress: () => void;
+};
+
+export const WalletItem = ({ device, isSelectable = true, onPress }: WalletItemProps) => {
```

Comment:
Optional parameters with default value should be last items in the spread according to code style